homeassistant/components/advantage_air/sensor.py

```diff
@@ -62,7 +65,7 @@ class AdvantageAirTimeTo(AdvantageAirEntity, SensorEntity):
     _attr_native_unit_of_measurement = ADVANTAGE_AIR_SET_COUNTDOWN_UNIT
     _attr_entity_category = EntityCategory.DIAGNOSTIC
 
-    def __init__(self, instance, ac_key, action):
+    def __init__(self, instance: advantage_air, ac_key: int, action: str) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3